recipe/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
     - python
     - google-api-core >=1.31.6,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
     - google-auth >=1.25.0,<3.0.0dev
+    - importlib-metadata>1.0.0  # [py<38]
     - grpcio >=1.38.0,<2.0.0dev
 
```

Comment:
Should `grpcio-status >= 1.38.0, < 2.0.dev0` also be added? 
https://github.com/googleapis/python-cloud-core/blob/4e695644185dc96e245e9fb38d8331ff33d6c16d/setup.py#L38